packages/next/build/webpack/plugins/middleware-plugin.ts

```diff
@@ -574,6 +589,35 @@ Learn More: https://nextjs.org/docs/messages/node-module-in-edge-runtime`,
   }
 }
 
+async function findEntryEdgeFunctionConfig(
+  entryDependency: any,
+  resolver: webpack.Resolver
+) {
+  if (entryDependency?.request?.startsWith('next-')) {
+    const absolutePagePath =
+      new URL(entryDependency.request, 'http://example.org').searchParams.get(
```

Comment:
noob question: What's the intention of 'http://example.org'? Why not `URL(entryDependency.request)`?

Reply:
`entryDependency.request` is a relative URL, so we need a dummy base path or we'll get an exception.